crates/router/src/routes/app.rs

```diff
@@ -666,6 +666,20 @@ impl BusinessProfile {
     }
 }
 
+pub struct Gsm;
+
+#[cfg(feature = "olap")]
+impl Gsm {
+    pub fn server(state: AppState) -> Scope {
+        web::scope("/gsm")
+            .app_data(web::Data::new(state))
+            .service(web::resource("").route(web::post().to(create_gsm_rule)))
+            .service(web::resource("/get").route(web::post().to(get_gsm_rule)))
+            .service(web::resource("/update").route(web::post().to(update_gsm_rule)))
+            .service(web::resource("/delete").route(web::post().to(delete_gsm_rule)))
```

Comment:
the endpoints and http methods do not match, ideally get should go with the GET http method, same for the rest

Reply:
For all CRUD APIs we need payload with connector, flow, sub_flow, code and message fields

Reply:
The GET endpoint can use query parameters if the values are all single values (and not a list of values).